app/src/main/java/de/westnordost/streetcomplete/util/LatLonRaster.kt

```diff
@@ -48,14 +48,14 @@ class LatLonRaster(bounds: BoundingBox, private val cellSize: Double) {
         val startY = max(0, min(latitudeToCellY(bounds.minLatitude), rasterHeight - 1))
         val endX = max(0, min(longitudeToCellX(bounds.maxLongitude), rasterWidth - 1))
         val endY = max(0, min(latitudeToCellY(bounds.maxLatitude), rasterHeight - 1))
-        val result = MultiIterable<LatLon>()
+        var result = sequenceOf<LatLon>()
         for (y in startY..endY) {
             for (x in startX..endX) {
                 val list = raster[y * rasterWidth + x]
-                if (list != null) result.add(list)
+                if (list != null) result += list
```

Comment:
Regarding the "+" operator - I can't find at all where this is defined. Does the plus operator do what we think it does even?

Reply:
Documentation's here: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.sequences/plus.html

These are the relevant definitions:

* https://github.com/JetBrains/kotlin/blob/8fa848bed384568a8ceb9731ec9d45663aa64521/libraries/stdlib/common/src/generated/_Sequences.kt#L2541-L2543
* https://github.com/JetBrains/kotlin/blob/8fa848bed384568a8ceb9731ec9d45663aa64521/libraries/stdlib/src/kotlin/collections/Sequences.kt#L82
* https://github.com/JetBrains/kotlin/blob/8fa848bed384568a8ceb9731ec9d45663aa64521/libraries/stdlib/src/kotlin/collections/Sequences.kt#L94-L99